<launch>
    <arg name="tello_slam_display_node_name" default="tello_slam_display_node"/>
    <arg name="flag_display_output_image" default="True"/>
    <arg name="display_output_image_window_name" default="tello_slam_display_window"/>

    <arg name="camera_image_topic_name" default="camera/camera_info"/>

    <arg name="image_topic_name" default="camera/image_raw"/>
    <arg name="output_image_topic_name" default="tello_slam/tello_slam_observation_image/image_raw" />

    <node name="$(arg tello_slam_display_node_name)" pkg="tello_slam_ros" type="tello_slam_display" output="screen">
        <!-- <param name="display_output_image_window_name" value="$(arg display_output_window_name)" type="string"/> -->
        <param name="image_topic_name" value="$(arg image_topic_name)" type="string"/>
        <param name="output_image_topic_name" value="$(arg output_image_topic_name)" type="string" />
        <param name="camera_image_topic_name" value="$(arg camera_image_topic_name)" type="string" />
    </node>
</launch>